<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgProveedores</class>
 <widget class="QWidget" name="wdgProveedores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblProveedores">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Proveedores</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblBusqueda">
         <property name="text">
          <string>Buscar proveedor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtBusqueda"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tblProveedores">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Id Proveedor</string>
        </property>
        <property name="textAlignment">
         <set>AlignRight|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Proveedor</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CIF</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Teléfono</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Correo electrónico</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Dirección</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ciudad</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Código Postal</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>País</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblAlbaran">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Albaranes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tblAlbaranes">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Id Albarán</string>
        </property>
        <property name="textAlignment">
         <set>AlignRight|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fecha</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Proveedor</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Artículos</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Coste</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Inventariado</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSoloAlbaranes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/factura.png</normaloff>:/gnuoptics/factura.png</iconset>
   </property>
   <property name="text">
    <string>Solo Albaranes</string>
   </property>
  </action>
  <action name="actionFiltrarInventariados">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/document-preview-archive.png</normaloff>:/gnuoptics/document-preview-archive.png</iconset>
   </property>
   <property name="text">
    <string>Filtrar los inventariados</string>
   </property>
  </action>
  <action name="actionProveedorNuevo">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/list-add-user.png</normaloff>:/gnuoptics/list-add-user.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo proveedor</string>
   </property>
  </action>
  <action name="actionProveedorBorrar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/list-remove.png</normaloff>:/gnuoptics/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar Proveedor</string>
   </property>
  </action>
  <action name="actionProveedorModificar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/document-edit.png</normaloff>:/gnuoptics/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Modificar Proveedor</string>
   </property>
  </action>
  <action name="actionAlbaranNuevo">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/list-add-user.png</normaloff>:/gnuoptics/list-add-user.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo albarán</string>
   </property>
   <property name="toolTip">
    <string>Nuevo albarán</string>
   </property>
  </action>
  <action name="actionAlbaranBorrar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/list-remove.png</normaloff>:/gnuoptics/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar albarán</string>
   </property>
   <property name="toolTip">
    <string>Borrar albarán</string>
   </property>
  </action>
  <action name="actionAlbaranModificar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/document-edit.png</normaloff>:/gnuoptics/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Modificar albarán</string>
   </property>
   <property name="toolTip">
    <string>Modificar albarán</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/gnuoptics.qrc"/>
 </resources>
 <connections/>
</ui>
